python/cudnn/gemm/cutedsl/grouped/dsrelu/api.py

```
@@ -34,14 +34,25 @@
+    default_stream,
+    detect_framework,
+    framework_dtype,
+    get_compute_capability,
+    get_data_ptr,
+    get_shape,
+    get_strides,
+    is_torch_tensor,
+)
 
 
 def _reinterpret_raw_grouped_fp4_tensor(tensor: torch.Tensor) -> torch.Tensor:
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)